tests/sentry/monitors/test_monitor_consumer.py

```diff
@@ -326,15 +326,20 @@ def test_check_in_empty_id_update(self):
         assert open_checkin.status == CheckInStatus.IN_PROGRESS
         assert open_checkin.guid != uuid.UUID(int=0)
 
+        # Send an event to a different monitor environment, tests that when we
+        # use the empty UUID "latest" we properly scope to the latest of the
+        # same monitor environment
+        self.send_message("my-monitor", status="in_progress", environment="dev")
```

Comment:
change this status to `ok`. otherwise it's only going to update the `date_updated` which we don't have any assertions on

Reply:
No this needs to be OK to correctly test test what this is guarding against.

This creates a NEW check-in in a different environment with the status `in_progress`

Reply:
I'll follow up and make the comment more clear. This one wasn't intending to assert that we didn't accidentally update the existing one, it was just setting it up so that there was a newer check-in.